src/ws_server.rs

```diff
@@ -30,8 +34,40 @@ impl WsServer {
 
 impl<T: Controller> Handler for WsHandler<T> {
 
-    fn on_open(&mut self, _: Handshake) -> Result<()> {
+    fn on_open(&mut self, handshake: Handshake) -> Result<()> {
         info!("Hello new ws connection");
+
+        let resource = &handshake.request.resource()[..];
+        let url = Url::parse(&format!("http://fox.box{}", resource)).unwrap();
```

Comment:
Also:

> [20:26:28]  <fabrice>	ferjm: Is that line hardcoding the host name? https://github.com/fxbox/foxbox/pull/167/files#diff-e1ea9eba222b6cc49837b3dd765989dfR41 
[20:26:57]  <ferjm>	fabrice no, it's just a random url
[20:28:00]  <ferjm>	fabrice he just need to build a random URL to get the query string cause handshake.request.resource only gives the path 
[20:28:13]  <ferjm>	at least that's how I understood the patch
[20:28:14]  <fabrice>	that's... well
[20:28:41]  <fabrice>	could use a comment because gross hack